### PR TITLE
Docs: Update getting_started.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
I am a new Prometheus user. The instructions confused me, because it is not clear that you need to click **Execute** in order to get or update results. Based on instructions, I expected that entering the expression would get me results. When pasting the text in did not work, I pressed Enter, and still no results. I only got results when I clicked **Execute**.

This is a minor change, but I think it will help first-time Prometheus users to understand how the interface works.

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -81,13 +81,12 @@ navigating to its metrics endpoint:
...
 use Prometheus's built-in expression browser, navigate to
-http://localhost:9090/graph and choose the "Console" view within the "Graph"
-tab.
+http://localhost:9090/graph and choose the "Console" view within the "Graph" tab.
 
 As you can gather from [localhost:9090/metrics](http://localhost:9090/metrics),
 one metric that Prometheus exports about itself is called
 `prometheus_target_interval_length_seconds` (the actual amount of time between
-target scrapes). Go ahead and enter this into the expression console:
+target scrapes). Go ahead and enter this into the expression console and then click **Execute**:
```
Comment (brian-brazil) [inline docs/getting_started.md]: We use double quotes for text on UI elements, not bold.

Comment (oddlittlebird) [reply docs/getting_started.md]: Updated.

**Comment by brian-brazil** [comment]
Can you add the DCO to the 2nd commit (or squash)?

**Comment by oddlittlebird** [comment]
@brian-brazil , done

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**